kolibri/plugins/setup_wizard/api.py

```diff
@@ -51,7 +51,9 @@ def createuseronremote(self, request):
         url = "{}{}".format(baseurl, api_url)
 
         payload = {
-            "facility_id": facility_id,
+            # N.B. facility is keyed by facility not facility_id on the signup
```

Comment:
What's `N.B.` mean here?

Why not update the serializer field to be `facility_id` instead?

Reply:
Too many other places to have to change on the frontend I suspect?

In any case, change looks good to me

Reply:
`Nota Bene` - note well.

Changing the public serializer would involve changing the serializer for all signups, so would be a fairly significant change. If I was going to make a more dramatic change here, I would propagate this back through this API endpoint (to be `facility` all the way) but this seemed like the least dramatic fix this close to release.